include/quda.h

```diff
@@ -1521,10 +1537,38 @@ extern "C" {
 
   /**
    * Computes the total, spatial and temporal plaquette averages of the loaded gauge configuration.
-   * @param Array for storing the averages (total, spatial, temporal)
+   * @param [out] Array for storing the averages (total, spatial, temporal)
```

Comment:
The doxygen here for the parameters, and in the lines below, has an erroneous space between `param` and `[in/out]`.  I haven't seen anywhere in the doxygen docs that this style is supported, so these spaces should be deleted.

Reply:
https://github.com/lattice/quda/commit/4a893d83c4c5b4c15c9e0d07f8cc3cd1a0e1955a

Reply:
https://github.com/lattice/quda/pull/1308/commits/59cf5f0abf8e0c5cb25ef657c81d520720b48439